packages/core/src/__tests__/memdb.test.ts

```diff
@@ -73,6 +73,38 @@ describe('memdb', () => {
     expect(multipleParam.length).toBe(10)
   })
 
+  it('should query model like params', async () => {
+    const hierarchy = new Hierarchy()
+    for (const tx of txes) hierarchy.tx(tx)
+    const model = new ModelDb(hierarchy)
+    for (const tx of txes) await model.tx(tx)
+    const expectedLength = txes.filter(tx => tx.objectSpace === core.space.Model).length
+    const without = await model.findAll(core.class.Doc, {
+      space: { $like: core.space.Model }
+    })
+    expect(without).toHaveLength(expectedLength)
+    const begin = await model.findAll(core.class.Doc, {
+      space: { $like: '*Model' }
+    })
+    expect(begin).toHaveLength(expectedLength)
+    const zero = await model.findAll(core.class.Doc, {
+      space: { $like: 'Model' }
+    })
+    expect(zero).toHaveLength(0)
+    const end = await model.findAll(core.class.Doc, {
+      space: { $like: 'space:core.M*' }
+    })
+    expect(end).toHaveLength(expectedLength)
+    const mid = await model.findAll(core.class.Doc, {
+      space: { $like: '*M*de*' }
```

Comment:
$like should allow only prefix and postfix?

Reply:
Allowed any position, why should we restrict the use? If we used only a prefix or only a postfix, then in the future it would be possible to optimize the algorithms (in particular for `MemDB`), but for a query like `*any*` it will not make sense

Reply:
ok, i've found sql support putting it inside. It also allow to match for multiple characters and to just one character.

https://dev.mysql.com/doc/refman/8.0/en/string-comparison-functions.html#operator_like